synapse/config/metrics.py

```diff
@@ -93,10 +93,11 @@ def generate_config_section(self, report_stats=None, **kwargs):
             #known_servers: true
 
         # Whether or not to report anonymized homeserver usage statistics.
+        #
         """
 
         if report_stats is None:
-            res += "# report_stats: true|false\n"
+            res += "#report_stats: true|false\n"
```

Comment:
The style guide seems to say that the example should provide the opposite of the default, but I see you just changed the formatting a bit here. :) So...still better.

Reply:
report_stats is odd, because it doesn't have a default.

Reply:
Ahh, I see. 👍